ext/web/lib.deno_web.d.ts

```diff
@@ -1241,20 +1322,27 @@ declare function reportError(
 /** @category Platform */
 declare type PredefinedColorSpace = "srgb" | "display-p3";
 
-/** @category Platform */
+/** Color space settings for `ImageData` objects.
+ *
+ *  @category Platform */
 declare interface ImageDataSettings {
   readonly colorSpace?: PredefinedColorSpace;
 }
 
-/** @category Platform */
+/** An object that holds information about an image.
+ *
+ * @category Platform */
 declare interface ImageData {
   readonly colorSpace: PredefinedColorSpace;
   readonly data: Uint8ClampedArray;
   readonly height: number;
   readonly width: number;
 }
 
-/** @category Platform */
+/** Work with image data, create new image data objects or manipulate existing
+ * pixel data.
```

Comment:
This is the same symbol as the interface above. Maybe doesn't need different doc comment